src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -87,9 +87,15 @@ public function load(array $configs, ContainerBuilder $container): void
         $config = $this->processConfiguration($configuration, $configs);
 
         $formats = $this->getFormats($config['formats']);
+        $patchFormats = $this->getFormats($config['patch_formats']);
         $errorFormats = $this->getFormats($config['error_formats']);
 
-        $this->registerCommonConfiguration($container, $config, $loader, $formats, $errorFormats);
+        // Backward Compatibility layer
+        if (isset($formats['jsonapi']) && !isset($patchFormats['jsonapi'])) {
```

Comment:
I really think we should deprecate specifying the format keys, and just use the mime type directly.